static/lang/re-en.json

```
@@ -5688,7 +5688,9 @@
                     "Chalice": "Chalice",
                     "Lantern": "Lantern",
                     "Mirror": "Mirror",
-                    "Regalia": "Regalia",
+                    "Regalia": {
+                        "UnflankableToggle": "Regalia — Origin is being flanked."
```

```suggestion
                        "UnflankableToggle": "Regalia owner is being flanked."
```
Maybe this is more obvious as to what it means. Origin has a system meaning that may trip up some users, since it could refer to the origin of the strike too.

---

Yeah, I'm cool with this